app/src/main/java/org/fossasia/openevent/adapters/FilterAdapter.java

```diff
@@ -0,0 +1,111 @@
+package org.fossasia.openevent.adapters;
+
+/**
+ * Created by user on 2/29/2016.
+ */
+import android.content.Context;
+import android.support.v7.widget.RecyclerView;
+import android.view.LayoutInflater;
+import android.view.View;
+import android.view.ViewGroup;
+import android.widget.CheckBox;
+import android.widget.TextView;
+
+
+import org.fossasia.openevent.R;
+import org.fossasia.openevent.utils.SaveFilters;
+
+import java.util.Collections;
+import java.util.List;
+
+@SuppressWarnings("deprecation")
+/**
+ * Created by user on 02-06-2015.
+ */
+public class FilterAdapter extends RecyclerView.Adapter<FilterAdapter.MyViewHolder> {
+    private LayoutInflater inflater;
+    Context context;
+    List<String> data = Collections.emptyList();
+
+    Boolean[] checkedFilters;
+
+
+    public FilterAdapter(Context context, List<String> data) {
+        inflater = LayoutInflater.from(context);
+        this.data = data;
+        this.context = context;
+        this.checkedFilters = new Boolean[data.size()];
+        this.checkedFilters = new SaveFilters().loadArray("Filters", context);
+
+    }
+
+    @Override
+    public MyViewHolder onCreateViewHolder(ViewGroup viewGroup, int i) {
+        View view = inflater.inflate(R.layout.filter_tiles, viewGroup, false);
+        MyViewHolder viewHolder = new MyViewHolder(view, context);
+        return viewHolder;
+    }
+
+    @Override
+    public void onBindViewHolder(MyViewHolder myViewHolder, int i) {
+        String tTitle = data.get(i);
+        myViewHolder.item.setText(tTitle);
+        myViewHolder.itemCheck.setChecked(checkedFilters[i]);
+
+
+       // myViewHolder.icon.setImageResource(current.itemId);
+
+
+    }
+
+
+    @Override
+    public int getItemCount() {
+        return data.size();
+    }
+
+
+    class MyViewHolder extends RecyclerView.ViewHolder implements View.OnClickListener {
+        TextView item;
```

Comment:
![Codacy](https://www.codacy.com/assets/images/favicon.png) Issue found: [Use explicit scoping instead of the default package private level](https://www.codacy.com/app/triplez/open-event-android/file/2406771676/issues/source?bid=3233850&fileBranchId=3233908#l69)